Makefile

```diff
@@ -2,10 +2,8 @@ PROJ = lkmpg
 all: $(PROJ).pdf
 
 $(PROJ).pdf: lkmpg.tex
-	pdflatex -shell-escap $<
-	bibtex $(PROJ) >/dev/null || echo
-	pdflatex -shell-escape $< 2>/dev/null >/dev/null
 	rm -rf _minted-$(PROJ)
+	latexmk -shell-escape lkmpg.tex -pdf
```

Comment:
Add the guard before invoking `latexmk`. That is,
```
@if ! hash latexmk; then echo "No Latexmk found. See https://mg.readthedocs.io/latexmk.html for installation."; exit 1; fi 
```

Reply:
Added as requested